### PR TITLE
Adjust API Exposure tutorials to the removal of serverless functions in version v1alpha1

### PR DESCRIPTION
**Pull request by nataliasitko**
Changes proposed in this pull request:

- In [Expose workloads in multiple Namespaces with a single APIRule definition](https://kyma-project.io/docs/kyma/main/03-tutorials/00-api-exposure/apix-04-expose-workload/apix-04-03-expose-workloads-multiple-namespaces/), simplify the prerequisites. Link to the Create a workload tutorial that contains the detailed instructions.
- Update the function.yaml file

### CODE REVIEW

**Review by strekm — DISMISSED** [review]
/meow

File: docs/03-tutorials/00-api-exposure/apix-04-expose-workload/apix-04-03-expose-workloads-multiple-namespaces.md [hunk 1]
```diff
@@ -8,45 +8,15 @@ This tutorial shows how to expose service endpoints in multiple Namespaces using
...
-     ```
-
-   * Create a Function in its Namespace using the [sample code](../assets/function.yaml):
-
-     ```bash
-     kubectl -n $NAMESPACE_FUNCTION apply -f https://raw.githubusercontent.com/kyma-project/kyma/main/docs/03-tutorials/assets/function.yaml
-     ```
-
-2. Create a Namespace for the Gateway and APIRule CRs and export its value as the environment variable. Run:
+1. Create three Namespaces: one for an instance of the HttpBin service, one for a sample function, and one for an APIRule CR. Deploy an instance of the HttpBin service and a sample Function in their respective Namespaces. To learn how to do it, follow the [Create a workload](../apix-01-create-workload.md) tutorial. Remember to enable the Istio sidecar proxy injection. Export the namespaces' names as environment variables:
```
Comment (grego952) [inline docs/03-tutorials/00-api-exposure/apix-04-expose-workload/apix-04-03-expose-workloads-multiple-namespaces.md:11]:
```suggestion
1. Create three Namespaces: one for an instance of the HttpBin service, one for a sample Function, and one for an APIRule custom resource (CR). Deploy an instance of the HttpBin service and a sample Function in their respective Namespaces. To learn how to do it, follow the [Create a workload](../apix-01-create-workload.md) tutorial. Remember to enable the Istio sidecar proxy injection. Export the Namespaces' names as environment variables:
```

File: docs/03-tutorials/00-api-exposure/apix-04-expose-workload/apix-04-03-expose-workloads-multiple-namespaces.md [hunk 2]
```diff
@@ -8,45 +8,19 @@ This tutorial shows how to expose service endpoints in multiple Namespaces using
 
 ##  Prerequisites
 
-1. Create a sample HttpBin service deployment and a sample Function in the separate Namespaces:
+1. Create three Namespaces: one for an instance of the HttpBin service, one for a sample Function, and one for an APIRule custom resource (CR). Deploy an instance of the HttpBin service and a sample Function in their respective Namespaces. To learn how to do it, follow the [Create a workload](../apix-01-create-workload.md) tutorial. 
 
-   * Create a Namespace for the HttpBin service and export its value as an environment variable. Run:
+  >**NOTE:** Remember to enable the Istio sidecar proxy injection.
```
Comment (grego952) [inline docs/03-tutorials/00-api-exposure/apix-04-expose-workload/apix-04-03-expose-workloads-multiple-namespaces.md:13]:
```suggestion
  >**NOTE:** Remember to [enable the Istio sidecar proxy injection](https://kyma-project.io/docs/kyma/latest/04-operation-guides/operations/smsh-01-istio-enable-sidecar-injection/).
```

**Review by grego952 — APPROVED** [review]

**Pull request merged**